pkg/services/ngalert/api/authorization_test.go

```diff
@@ -40,7 +40,7 @@ func TestAuthorize(t *testing.T) {
 		}
 		paths[p] = methods
 	}
-	require.Len(t, paths, 64)
+	require.Len(t, paths, 65)
```

Comment:
This should have been bumped, but the test is failing.

Reply:
The output looks like this: 
```/ruler/grafana/api/v1/rules/{Namespace}:[GET POST DELETE] /ruler/grafana/api/v1/rules/{Namespace}/export:[POST] /ruler/grafana/api/v1/rules/{Namespace}/{Groupname}:[GET DELETE]```, and this change only adds a new method, not a new path. I wouldn't expect this number to increase for this change.